home/src/main/webapp/roomType.js

```diff
@@ -1,164 +1,88 @@
 $(function () {
-  let n = 1
-  const roomTypes =
-    {
-      0: {},
-      A: {
-        "name": "單人房A",
-        "chair": "椅子*1",
-        "bed": "雙人床",
-        "desk": "大桌子",
-        "sideTable": "小床頭櫃",
-        "wardrobe": "衣櫃",
-        "size": 5,
-        "price": 8100,
-        "balcony": false,
-        "pic": ["SingleA1.jpg", "SingleA2.jpg", "SingleA3.jpg"],
-        "rest": 1,
-        "availableFloor": [1, 3, 8]
-      },
-      B: {
-        "name": "單人房B",
-        "chair": "椅子*1",
-        "bed": "雙人床",
-        "desk": "大桌子",
-        "sideTable": "小床頭櫃",
-        "wardrobe": "衣櫃",
-        "size": 5.2,
-        "price": 8200,
-        "balcony": true,
-        "pic": ["SingleB1.jpg", "SingleB2.jpg", "SingleB3.jpg"],
-        "rest": 2,
-        "availableFloor": [1, 5, 8]
-      },
-      C: {
-        "name": "單人房C",
-        "chair": "椅子*2",
-        "bed": "單人床",
-        "desk": "大桌子",
-        "sideTable": "小床頭櫃",
-        "wardrobe": "大衣櫃",
-        "size": 5.3,
-        "price": 8300,
-        "balcony": true,
-        "pic": ["SingleC1.jpg", "SingleC2.jpg", "SingleC3.jpg"],
-        "rest": 3,
-        "availableFloor": [1, 2]
-      },
-      D: {
-        "name": "雙人房A",
-        "chair": "椅子*2",
-        "bed": "單人床",
-        "desk": "大桌子",
-        "sideTable": "小床頭櫃",
-        "wardrobe": "衣櫃",
-        "size": 6.1,
-        "price": 8400,
-        "balcony": false,
-        "pic": ["DoubleA1.jpg", "DoubleA2.jpg", "DoubleA3.jpg"],
-        "rest": 4,
-        "availableFloor": [4, 5]
-      },
-      E: {
-        "name": "雙人房B",
-        "chair": "椅子*2",
-        "bed": "單人床",
-        "desk": "大桌子",
-        "sideTable": "小床頭櫃",
-        "wardrobe": "衣櫃",
-        "size": 6.2,
-        "price": 8500,
-        "balcony": true,
-        "pic": ["DoubleB1.jpg", "DoubleB2.jpg", "DoubleB3.jpg"],
-        "rest": 5,
-        "availableFloor": [8]
-      },
-      F: {
-        "name": "雙人房C",
-        "chair": "椅子*2",
-        "bed": "單人床",
-        "desk": "大桌子",
-        "sideTable": "小床頭櫃",
-        "wardrobe": "衣櫃",
-        "size": 6.3,
-        "price": 8600,
-        "balcony": true,
-        "pic": ["DoubleC1.jpg", "DoubleC2.jpg", "DoubleC3.jpg"],
-        "rest": 0,
-        "availableFloor": []
-      }
-    }
 
-  function renderFeature(n) {
-    const roomTypeKeys = Object.keys(roomTypes)[n]
-    const roomType = roomTypes[roomTypeKeys];
+  const Type = "A"  //begin type ,會由nav產生
+  let asciiType = Type.charCodeAt()   //A ~ F =65 ~70
+  function renderFeature(asciiType) {
+    const type = String.fromCharCode(asciiType)
 
-    $('#room-type-title').text(roomType.name)
-    $('#room-type-price').text(roomType.price + " 元 / 月")
-    $('#room-type-rest').text(roomType.rest)
+    $.getJSON('/home/common/RoomTypeServlet', type, function (res) {
```

Comment:
```suggestion
    $.getJSON(`/home/common/RoomTypeServlet?roomType=${type}`, function (res) {
```

並搭配 Servlet `request.getParameter("roomType");` 取出對應的 type